test/e2e/Tests/Tests/PurgeInstancesTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E;
+
+[Collection(Constants.FunctionAppCollectionName)]
+public class PurgeInstancesTests
+{
+    private readonly FunctionAppFixture _fixture;
+    private readonly ITestOutputHelper _output;
+
+    public PurgeInstancesTests(FunctionAppFixture fixture, ITestOutputHelper testOutputHelper)
+    {
+        _fixture = fixture;
+        _fixture.TestLogs.UseTestLogger(testOutputHelper);
+        _output = testOutputHelper;
+    }
+
+    [Fact]
+    public async Task PurgeOrchestrationHistory_StartAndEnd_Succeeds()
+    {
+        DateTime purgeStartTime = DateTime.MinValue;
+        DateTime purgeEndTime = DateTime.UtcNow;
+        string queryParams = $"?purgeStartTime={purgeStartTime.ToString("o")}&purgeEndTime={purgeEndTime.ToString("o")}";
+        using HttpResponseMessage response = await HttpHelpers.InvokeHttpTrigger("PurgeOrchestrationHistory", queryParams);
+        string actualMessage = await response.Content.ReadAsStringAsync();
+        Assert.Matches(@"^Purged [0-9]* records$", actualMessage);
```

Comment:
Looks like we may or may not have any actual data that gets purged in this test and in most of the other tests. How should I think about what "success" means in these cases?